.github/workflows/action.yml

```diff
@@ -20,3 +20,5 @@ jobs:
         title: Link Checker Report
         content-filepath: ./link-checker/out.md
         labels: links
+    - name: Fail if there were link errors
+      run: exit ${{ steps.lc.outputs.exit_code }}
```

Comment:
Also need to set the `lc` id to the `peter-evans/link-checker@v1` step: https://github.com/peter-evans/link-checker#creating-a-failing-check-for-link-errors

Reply:
Set in https://github.com/microbit-foundation/dev-docs/pull/230/commits/f89315c59822ca5911c06a082425b162e10705d7